packages/mdc-list/_mixins.scss

```diff
@@ -78,6 +78,11 @@
     }
   }
 
+  // Only enable mouse hover for interactive list items which are not disabled.
+  :not(.mdc-list--non-interactive) > .mdc-list-item:not(.mdc-list-item--disabled) {
```

Comment:
I'd be inclined to match the selector we use in the `mdc-list-ripple` mixin here, which also has `.mdc-list-item` on the last compound selector (to avoid applying this to every child of the list, which could include other elements such as dividers).

Reply:
Changed the order in the other selector instead, as discussed.

Reply:
Switched back to original order (and updated here to be consistent) with .mdc-list-item last, since we use &--activated, etc. in `mdc-list-ripple`, forcing .mdc-list-item to go last.